gradle.properties

```diff
@@ -2,4 +2,5 @@ version=2.6.1-SNAPSHOT
 org.gradle.caching=true
 org.gradle.parallel=true
 kafkaVersion=3.3.1
-confluentVersion=7.3.0
+confluentVersion=7.3.1
+org.gradle.jvmargs=-Xmx2048m
```

Comment:
Whats the reason for setting this jvmarg here as a default?

Reply:
Sonarqube crashed with the Gradle update